### ISSUE TITLE
Bug in Firefox: The route does not exist and other js errors

### ISSUE THREAD

**Issue by pardahlman** [akeneo/pim-community-dev#6044]
Hello team 👋 ,

Thanks for an amazing system! I'm running version 1.7.0 Akeneo on Linux, here's the generated
[system info file](https://github.com/akeneo/pim-community-dev/files/905333/akeneo-pim-system-info_2017-04-07_12-55.txt).

I'm using the web interface on my Windows 10 machine, and noticed that many of the views aren't rendered correctly on  Firefox (latest version, 52.0.1).

For example, there is an error message in the console when I navigate to `/configuration/channel/` 

```
> Error: The route "pim_datagrid_view_rest_default_columns" does not exist.   router.js:9:313
```
If I navigate to `/configuration/attribute/` I get the following error

```
> TypeError: _.__ is not a function[Learn More]  abstract-filter.js:61:22
```

This seams to abort the rendering of the site, leaving the main content are blank, see screen shot below

![image](https://cloud.githubusercontent.com/assets/1032755/24798615/281ab6ac-1b97-11e7-86b7-6cf6e011b904.png)

When I visit the same URL with latest version of Chrome (57.0.2987.133) the views are rendered as expected.

**Comment by mickaelandrieu** [comment]
Hello @pardahlman,

looking at your screenshot, the url should be `#url=/configuration/channel/` isn't it?

I can't reproduce the issue on `1.7.4` // `Firefox 54` // `Windows 10`, would you mind to update your PIM and confirm the issue is fixed?

Thank for your report?

**Comment by pardahlman** [comment]
Hey, @mickaelandrieu - thanks for getting back to me.

I think you are right, I tried to reproduce the issue in my current instance running 1.7.3 CE, and I get the same URL as you and it works 👍 

I have a different issue on the same theme, though... I'm not able to list products in Firefox, see screenshot below

![products-1 7 3](https://cloud.githubusercontent.com/assets/1032755/26103369/526dc9d8-3a39-11e7-9ffb-42c1cf016940.png)

The console log states: "Empty string passed to getElementById().". Tried to reproduce this on demo.akeneo.com but it works there. Perhaps it something with my installation, but it works in Chrome which feels strange to me.

**Comment by mickaelandrieu** [comment]
@pardahlman I was not able to reproduce on my own side both on Chrome and Firefox... hmm is there something special with your products, can you try to make a reproducible test case? Something we can reproduce on demo.akeneo.com could be really useful to help us fix the issue :)

**Comment by pardahlman** [comment]
I know, very frustrating. There is nothing special with the products, expect that they (and the categories) may contain some special characters (å, ä and ö), but that shouldn't fail. I've tried to repo it in demo, but without any success.

Also, since the problem is browser specific I just get the feeling that is has to do with some corner case in the javascript... but then again, it worked at the demo, so... I don't. I've tried to dig into the problem a bit further, but I didn't really find anything useful, except the fact that the value of `selector` is just `#`, and the call to `getElementById` is performed by an empty string. Here's my debugger session

![image](https://cloud.githubusercontent.com/assets/1032755/26110493/d083dd44-3a52-11e7-9bdb-dc03e63378c1.png)

It is perhaps best to close this issue, and I'll be sure to file a new one if I have some more concrete information about the issue.
